halconfig/settings.js

```diff
@@ -232,6 +232,7 @@ window.spinnakerSettings = {
     },
   },
   pubsubProviders: ['google'], // TODO(joonlim): Add amazon once it is confirmed that amazon pub/sub works.
+  plugins: '{{%plugins%}}',
```

Comment:
@link108 didn't we have an issue with this when we were looking at it from the Halyard side of things?

Reply:
This is from the original pr. 
I’m going to lean on your expertise to make the halyard bits work since we don’t use halyard at Netflix

Reply:
This is from the original pr. 
I’m going to lean on your expertise to make the halyard bits work since we don’t use halyard at Netflix